src/vs/base/browser/ui/menu/menubar.css

```diff
@@ -23,13 +24,18 @@
 	align-items: center;
 	box-sizing: border-box;
 	padding: 0px 8px;
+	border-radius: 5px;
 	cursor: default;
 	-webkit-app-region: no-drag;
 	zoom: 1;
 	white-space: nowrap;
 	outline: 0;
 }
 
+.menubar-menu-title {
+	line-height: 22px;
```

Comment:
I can't figure out the purpose of this rule. Removing it doesn't seem to have any negative effect